app/assets/stylesheets/blacklight.scss

```diff
@@ -66,3 +66,37 @@
     }
   }
 }
+
+dt.blacklight-full_text_tesimv {
+  @extend .h4;
+  display: inline-block;
+  float: none;
+  text-align: initial;
+  width: auto;
+
+  &[data-toggle="collapse"]:after {
+    /* symbol for "opening" panels */
+    font-family: 'Glyphicons Halflings';  /* essential for enabling glyphicon */
+    content: "\e114";    /* adjust as needed, taken from bootstrap.css */
+    float: right;        /* adjust as needed */
+    color: grey;         /* adjust as needed */
+    font-size: 0.8em;
+    line-height: normal;
+    padding-left: 2ch;
+  }
+
+  &.collapsed:after {
+    /* symbol for "collapsed" panels */
+    content: "\e080";    /* adjust as needed, taken from bootstrap.css */
+  }
+}
+
+dd.blacklight-full_text_tesimv {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector `blacklight-full_text_tesimv` should be written in lowercase with hyphens